base/server/src/main/java/com/netscape/cmscore/usrgrp/UGSubsystem.java

```diff
@@ -77,6 +77,7 @@ public class UGSubsystem {
     protected static final String LDAP_ATTR_CERTDN = "seeAlso";
     protected static final String LDAP_ATTR_USER_CERT = "userCertificate";
     protected static final String LDAP_ATTR_PROFILE_ID = "profileID";
+    protected static final String LDAP_ATTR_NS_PAGED_SIZE_LIMIT = "20000";
```

Comment:
I will remove this when I fix the broken test, but just pushing solution for now to see if the CI likes it (and the reviewers, of course :+1: )